src/simulator_core/entities/assets/heatexchanger.py

```diff
@@ -31,9 +32,9 @@ def __init__(
             pandapipes_net: pandapipesNet,
             diameter_m: float,
```

Comment:
Misschien moeten we hier ook gewoon een standaard diameter invoeren? Dan hoeven we dat bij het gebruik niet meer te doen: uiteindelijk niet relevant voor ons?

Reply:
Default diameter toegevoegd